src/main/io/rc_controls.h

```diff
@@ -109,10 +109,14 @@ typedef enum {
 #define MAX_MODE_RANGE_STEP ((CHANNEL_RANGE_MAX - CHANNEL_RANGE_MIN) / 25)
 
 // Roll/pitch rates are a proportion used for mixing, so it tops out at 1.0:
-#define CONTROL_RATE_CONFIG_ROLL_PITCH_RATE_MAX  100
+#define CONTROL_RATE_CONFIG_ROLL_PITCH_RATE_MAX  180
```

Comment:
Maybe add a comment with units for these values? This is not dps and that's confusing. Took me a while to figure out - in fact I was about to write a comment about value being too low :smile: